cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1694,13 +1746,20 @@ pub fn define(
         for to_type in ValueType::all_lane_types().filter(|t| t.lane_bits() >= 8 && *t != from_type)
         {
             let instruction = raw_bitcast
-                .bind_vector(to_type, 128 / to_type.lane_bits())
-                .bind_vector(from_type, 128 / from_type.lane_bits());
+                .bind_vector_from_lane(to_type, sse_vector_size)
+                .bind_vector_from_lane(from_type, sse_vector_size);
             e.enc32_rec(instruction.clone(), rec_null_fpr, 0);
             e.enc64_rec(instruction, rec_null_fpr, 0);
         }
     }
 
+    // SIMD vconst using MOVUPS
```

Comment:
Can you also comment that we use `MOVUPS` because we don't currently have a way to align the constant pools?